### PR TITLE
cloudflare.sh: Fix the script and add verification

### PR DESCRIPTION
**Pull request by jsarenik**
Fixes https://github.com/spf-tools/spf-tools/issues/164

Adds support for basic token verification: If run with

    ./cloudflare.sh verify

The only needed permissions for a custom API token are:
  - Zone.Zone: Read
  - Zone.DNS: Edit

### CODE REVIEW

**Comment by jsarenik** [comment]
@jcbf @AlexCline Review please?

**Comment by AlexCline** [comment]
LGTM

https://api.cloudflare.com/#user-api-tokens-verify-token

File: README.md [hunk 1]
```diff
@@ -172,6 +172,10 @@ and `ORIG_SPF` variables which stand for the target SPF domain
 (e.g. `spf-orig.spf-tools.eu.org`) in order to use `runspftools.sh`
 without modifying the script.
 
+The only needed permissions for a custom API token are:
```
[the remark this answers is not in the document]
Comment (jsarenik) [reply README.md:177]: Added in 256d8993c40f2a27301f629e4b6d309c6b6f0910

**Comment by jsarenik** [comment]
@jcbf and @AlexCline thank you for feed-back!!! I really apreciate it.

**Pull request merged**